app/controllers/concerns/fragment_cachable.rb

```diff
@@ -37,6 +37,6 @@ def foi_fragment_cache_write(key_path, content)
     File.atomic_write(key_path) do |f|
       f.write(content)
     end
-    FileUtils.chmod 0644, key_path
+    FileUtils.chmod 0o644, key_path
```

Comment:
Hmm I don't quite understand this check but -1 without exploring this more.

`FileUtils.chmod 0644` is much clearer than `FileUtils.chmod 0o644` to me.

Some of the changes where we're parsing dates don't look right.

Reply:
The commit has been removed, and the cop disabled (for now).